orttraining/orttraining/python/training/_ortmodule_suppress_output.py

```diff
@@ -0,0 +1,25 @@
+import os
+import sys
+from contextlib import contextmanager
+
+
+@contextmanager 
+def suppress_output(suppress_stdout=True,suppress_stderr=True, f=os.devnull):
```

Comment:
Can this be integrated with the logger settings we already have at https://github.com/microsoft/onnxruntime/blob/master/orttraining/orttraining/python/training/ortmodule.py#L286?